### ISSUE TITLE
Drag & Drop songs when playlist is not open

### ISSUE THREAD

**Issue by timea-techgirl** [digimezzo/dopamine-windows#1021]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report  
[X] Feature request
</code></pre>

## Feature
**Drag and drop a song to dopamine to play. Dopamine doesn't allow me to drag and drop songs to the window. It allows only if the playlist box is open. Can you allow drag and drop without opening the playlist box please? I have to open the playlist box each time.**

**Comment by kimuk1** [comment]
Yes I am trying to drag and drop songs, plus signs is shown with "copy" label but nothing imports.  Also any chance of windows right click intergration?
